backend/migrations/000021_add_achievements_details_column.up.sql

```diff
@@ -1,3 +0,0 @@
-ALTER TABLE community_users_achievements ADD details VARCHAR NOT NULL;
-ALTER TABLE community_users_achievements ADD UNIQUE (details);
-ALTER TABLE community_users_achievements RENAME TO user_achievements;
```

Comment:
dup of https://github.com/DapperCollectives/CAST/blob/375d108e3ef2b3cd0d11b2d8772ad295e8e6f6bf/backend/migrations/000020_add_achievements_details_column.up.sql#L0-L1